tests/upgrade_script/static/key_mapping_empty.log

```diff
@@ -0,0 +1,22 @@
+[INFO]    Mapping deployment into:
+[INFO]    Removing deployment
+[INFO]    Removing eventsDeployment
+[INFO]    Mapping fluentd.autoscaling into:
```

Comment:
What happened to this `autoscaling`?

Reply:
It should be object instead of string, but it's not the case of this test